third_party/maya/lib/pxrUsdMayaGL/batchRenderer.h

```diff
@@ -217,6 +219,13 @@ class UsdMayaGLBatchRenderer : public TfSingleton<UsdMayaGLBatchRenderer>
             const GfMatrix4d& projectionMatrix,
             GfVec3d* hitPoint);
 
+    /// Gets the UsdMayaGLSoftSelectHelper that this batchRenderer maintains.
```

Comment:
Looks like this docstring needs to be updated, since this returns a bool and is also used outside of PxrMayaHdShapeAdapter (in proxyShapeDelegate.cpp)